### ISSUE TITLE
TF 2.4.2, No AVX (nehalem arch), no GPU, Ubuntu 16.04, Python 3.7

### ISSUE THREAD

**Issue by Robbi373** [yaroslavvb/tensorflow-community-wheels#195]
Information is in the title, here ya go:

https://drive.google.com/file/d/11UUWu3O01K25sD4usyroSzEtW6Cg2Sb9/view?usp=sharing

Has been verified to work, see #123

**Comment by joshwand** [comment]
Does not WFM on Ubuntu 16.04.3 LTS, Atom E3930 (aws deeplens 1.0). Installs fine but when running I get this:

```
  File "/home/aws_cam/.local/lib/python3.7/site-packages/tensorflow/python/pywrap_tensorflow.py", line 64, in <module>
    from tensorflow.python._pywrap_tensorflow_internal import *
ImportError: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by /home/aws_cam/.local/lib/python3.7/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)
```

**Comment by Robbi373** [comment]
This is weird, since the Ubuntu 16.04 docker I used has glibc 2.23 installed, I don't know where this 2.27 requirement comes from. Did you by any chance install a 18.04-aimed tensorflow wheel beforehand? Since glibc 2.27 is default for 18.04, as far as I know.
You can force the reinstall by `pip install --force-reinstall --no-cache-dir {wheel name}.`

**Comment by joshwand** [comment]
You're right, I did try installing a 18.04 package. I tried reinstalling as above, but still getting the same error. Tried completely wiping tensor* from site-packages, but same result :(

**Comment by joshwand** [comment]
Oops, I was pointing at the wrong .whl the whole time! Please disregard :)
